.github/workflows/discord_PR_notify.yml

```diff
@@ -14,7 +16,7 @@ jobs:
     steps:
       - name: Discord Notification
         env: 
-          DISCORD_WEBHOOK: https://discord.com/api/webhooks/939296836240429116/m7GgUOoMyP8II1sKr9Vt1BFwJ2RUhNgzbZ-7hsoSKGzsJMMY69IR8UY35hGXKMhWtcVH
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
```

Comment:
This is cool. I'd love to see how you implemented this. Do you know how to use secrets for our environmental vars?

Reply:
You can check out the secrets in the repo or org settings. This one is an org secret. I think maybe we could use them for the airtable secrets but we should look more into the security.